cli/types/cypress.d.ts

```diff
@@ -5825,13 +5826,25 @@ declare namespace Cypress {
   }
 
   interface Cookie {
-    name: string
-    value: string
-    path: string
-    domain: string
-    httpOnly: boolean
-    secure: boolean
-    expiry?: number
+    /** The name of the cookie. */
+    name: string;
+    /** The value of the cookie. */
+    value: string;
+    /** Optional. The expiration date of the cookie as the number of seconds since the UNIX epoch. Not provided for session cookies.  */
+    expirationDate?: number;
+    /** True if the cookie is marked as HttpOnly (i.e. the cookie is inaccessible to client-side scripts). */
+    httpOnly: boolean;
+    /** The domain of the cookie (e.g. "www.google.com", "example.com"). */
+    domain: string;
+    /** The path of the cookie. */
+    path: string;
+    /** True if the cookie is marked as Secure (i.e. its scope is limited to secure channels, typically HTTPS). */
+    secure: boolean;
+    /** True if the cookie is a host-only cookie (i.e. a request's host must exactly match the domain of the cookie). */
+    hostOnly: boolean;
+    /**
+     * The cookie's same-site status (i.e. whether the cookie is sent with cross-site requests).
```

Comment:
Ripped this suggestion from MDN. A cookies same-site status doesn't mean a cookie will/won't be sent with a cross-site request. 
```suggestion
     * The cookie's same-site status (i.e. whether the cookie should be restricted to a first-party or same-site context).
```